sympy/core/power.py

```diff
@@ -1661,6 +1661,8 @@ def _eval_as_leading_term(self, x, logx=None, cdir=0):
             from sympy.functions.elementary.complexes import im
             try:
                 f = b.as_leading_term(x, logx=logx, cdir=cdir)
+                c, d = f.as_coeff_exponent(x)
+                lt=self.func(c, e)*self.func(x**d, e)
```

Comment:
spaces on either side of the assignment operator

Reply:
noted, will incorporate this change along with any other reviews/suggestions in future in the next commit.